spec/PostgresStorageAdapter.spec.js

```diff
@@ -1,6 +1,17 @@
 const PostgresStorageAdapter = require('../src/Adapters/Storage/Postgres/PostgresStorageAdapter');
 const databaseURI = 'postgres://localhost:5432/parse_server_postgres_adapter_test_database';
 
+const getColumns = (client, className) => {
```

Comment:
This whole method can be implemented with a single line, using method [map](http://vitaly-t.github.io/pg-promise/Database.html#map).

Reply:
This is a pretty simple one. We should setup map and save some lines :wink:. You've already done quite a lot so let me know if you can get around to this @paulovitin . If not I can put in an additional commit myself.